Shared/Samples/Change map view background/ChangeMapViewBackgroundView.Model.swift

```diff
@@ -12,9 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import Foundation
-import SwiftUI
 import ArcGIS
+import SwiftUI
```

Comment:
This model uses `Color` which unfortunately relies on SwiftUI. Nevertheless, since SwiftUI implicitly depend on Foundation, I removed it.

Reply:
While it does use `Color`, it looks like it just converts all colors to `UIColor`, so the code might be simpler if it used `UIColor` instead. That would still mean importing a UI framework (`UIKit` instead of `SwiftUI`), but that can't really be avoided in this case.

Reply:
Seems like the [ColorPicker](https://developer.apple.com/documentation/swiftui/colorpicker) in the other file/view which consumes these properties only supports `Color` or `CGColor` as initializing parameters, so I'll keep it unchanged for now.